help/en/html/tools/Blocks.shtml

```diff
@@ -104,12 +104,10 @@
 
       <p>Blocks are kept in your <i>layout configuration</i>, along
       with Turnouts, Sensors, Signal Heads, control panel setup etc.
-      To store this information on disk, allowing to
+      To store this information on disk, allowing you to
       <a href="../../package/jmri/jmrit/display/PanelMenuHelp.shtml">reload
-      it</a> next time you run JMRI, use <b>Store Configuration...</b>
-      in the <b>File</b> menu at the top of the Block Table (or other
-      tables from the Tools menu), or select <b>Store Panel...</b>
-      in the <b>Panel</b> menu. Note that the occupied/unoccupied
+      it</a> next time you run JMRI, see <a href="../apps/LoadStoreWork.shtml">
+      Loading and Storing Your Work</a>.  Note that the occupied/unoccupied
```

Comment:
I see that you have taken the specific instruction out of each table’s help page and referred people to LoadStoreWork.  I’m a bit in favor of the “belt and suspenders” approach of leaving the specific command sequence in the help and referring them to LoadStoreWork for more info.